portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/Form.path

```diff
@@ -587,13 +587,18 @@
 </tr>
 <!--PAGE-->
 <tr>
-	<td>PAGE_MOVE_ACTIVATED</td>
-	<td>//input[contains(@class,'page-header-title') and @value='${key_fieldName}']/../../../../..//button[contains(@class,'page-button') and @title='${key_movePage}']</td>
+	<td>PAGE_MOVE_UP_ENABLE</td>
+	<td>//input[contains(@class,'page-header-title') and @value='${key_formPageName}']/../../../../..//button[contains(@class,'page-button') and @class='reorder-page-button btn btn-monospaced btn-sm btn-secondary'][1]</td>
```

Comment:
@leticiachiu, do you think that we can look for the class `reorder-page-button` only at the end of this path?

The other CSS classes aren't maintained by our team, hence they're not under our control which can make the developers from our team remove them in the future due to changes introduced by who maintain them. Verifying only the class `reorder-page-button` can be a good solution in the long term. :slightly_smiling_face: